lib/simulation/openfisca/mapping/index.js

```diff
@@ -40,14 +40,17 @@ var applyRessources = function(src, dest, schema, dateDeValeur) {
 
     var ressourcesByType = _.groupBy(src.ressources, 'type');
     _.forEach(schema, function(definition, destKey) {
-        var srcKey, fn;
+        var srcKeys, fn;
         if (_.isString(definition)) {
-            srcKey = definition;
+            srcKeys = [definition];
+        } else if(_.isArray(definition)) {
+            srcKeys = definition;
         } else {
-            srcKey = definition.src;
+            srcKeys = [definition.src];
             fn = definition.fn;
         }
 
+    var mapRessources = function(srcKey) {
```

Comment:
Arg, JS is still not super clear for me, so I'm a bit confused. 

I don't have time to work on it more tonight, so I could work on it quickly tomorrow morning, but if we don't want to waste too much time, maybe you can do it ? 
Thanks ! 
Florian